lib/doorkeeper/config.rb

```diff
@@ -322,6 +332,18 @@ def raise_on_errors?
       handle_auth_errors == :raise
     end
 
+    def hash_secrets?
+      !@secret_hashing.nil?
+    end
+
+    def optional_hash(plain_token)
```

Comment:
This naming is not fully clear to me.. Maybe we can rename it to something more transparent? I don't know, like `encrypted_or_plain_token(token)` (if I understood correctly the behavior of this method)

Reply:
The method will return the hashed plaintext token value IFF token hashing is enabled, otherwise it will return the plain value.

In the same spirit as the other comments, I would suggest we stick to the name `hashing` (vs encryption). What about `hashed_or_plain_token(plain_token)`  then?

Reply:
Ok, so we'' work with hashes. Yes, `hashed_or_plain_token` would be better :+1: